package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "dialog-polyfill",
+  "version": "0.3.0",
```

Comment:
I'm about 99.9% sure NPM actually needs the version string.  Otherwise it complains. Plus, it caches based on this string.